glean_parser/schemas/metrics.2-0-0.schema.yaml

```diff
@@ -0,0 +1,591 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+---
+$schema: http://json-schema.org/draft-07/schema#
+title: Metrics
+description: |
+  Schema for the metrics.yaml files for Mozilla's Glean telemetry SDK.
+
+  The top-level of the `metrics.yaml` file has a key defining each category of
+  metrics. Categories must be snake_case, and they may also have dots `.` to
+  define subcategories.
```

Comment:
Is it worth capturing the key differences with the previous schema as part of the description? (I'm not sure)

Reply:
That's a good idea.  The key differences are in the `HISTORY.rst` entry, but documenting the differences directly in the new schemas seems like it would also be useful.